### PR TITLE
Metazoa config for Release e112

### PR DESCRIPTION
**Pull request by ens-LCampbell**
## Description

Changes to allowed species, biomart and introduction of pan genome for drosophila. 

**Related JIRA tickets:**
https://www.ebi.ac.uk/panda/jira/browse/ENSINT-1649

## Overview of changes
allowed_species.json, biomart_species.json, mlss_conf.xml 

#### Change 1
Introduction of multiple new species and some existing species of drosophila sp. Listed as part of the pangenome_drosophila in mlss_conf.xml. 36 species of drosophila total, no change to _D. melanogaster_ made! 

Includes four outgroup species

- bactrocera_neohumeralis_gca024586455v2rs
- coremacera_marginata_gca914767935v1
- machimus_atricapillus_gca933228815v1
- myopa_tessellatipennis_gca943737955v1
- 

#### Change 2
Intorduction of new configuration file for metazoa - _biomart_species.json_. 
Previously biomart was generated from all  metazoan species. Now restricted to 266 species includes new species as of e112, which is still less than the cap of 275 species. 
Removed older species of drosphila currently on ensembl main, those species all now replaced with new assemblies.
All species present in biomart set also in allowed_species.json

_drosophila_yakuba_ retained in allowed_spcies but removed from biomart in place of drosophila_yakuba_gca016746365v2rs. Potentially needs dropped entirely, though not discussed yet.

#### Change 3
Added new species of Crustacean lobster homarus_gammarus_gca958450375v1 into metazoa clade set

#### Change 4
Added new species of Tardigrade paramacrobiotus_metropolitanus_gca019649055v1rs  into metazoa and protostome clade sets.

#### Change 5
Added new species of Mollusc scallop P. maximus  into protostome clade set.

#### Change 6
Added new species of Echinoderm L. pictus  into metazoa clade set.

#### Change 7
Updated O. biamaculoides octopus to newer assembly version octopus_bimaculoides_gca001194135v2rs to both metazoa and protostome clade sets. 

#### Change 8
Updated tick R. sanguinesis to newer assembly  rhipicephalus_sanguineus_gca013339695v2rs to both metazoa and protostome clade sets. 

## Testing
Not tested

## Notes
Potentially one more change to bombus_terrestris required, still deliberating if we can make this change as this species is in alternate compute sets metazoa vs insects. Change will come in later PR if approved. 

---

For code reviewers: [code review SOP](https://www.ebi.ac.uk/seqdb/confluence/display/EnsCom/Code+review+SOP)

### CODE REVIEW

**Comment by ens-LCampbell** [comment]
> Hi @ens-LCampbell.. It looks like this PR would replace `octopus_bimaculoides` by `octopus_bimaculoides_gca001194135v2rs` in Metazoa, but the former is currently one of the Pan Metazoa species ([here](https://github.com/Ensembl/ensembl-compara/blob/dd7c140a995dc1f668caa889cf57c8af15723343/conf/pan/additional_species.json#L137)). Would you have in mind to replace `octopus_bimaculoides` by `octopus_bimaculoides_gca001194135v2rs` in Pan Compara too?

Thanks Thomas, my mistake to not check PanCompara also. I would be content changing to the newer Octopus assembly but Id like to run this past Jorge and Fergal. PanCompara should be stable as possible is my understanding. I image the update would be fine but id like to check. I will follow up Monday morning thanks

**Comment by ens-LCampbell** [comment]
Update MLSS and JSON to include the newer B. terrestris _bombus_terrestris_gca910591885v2_. Retaining the older assembly _bombus_terrestris_ as well.

File: conf/metazoa/mlss_conf.xml [hunk 1]
```diff
@@ -298,12 +304,64 @@
...
+      <genome name="coremacera_marginata_gca914767935v1"/>
+      <genome name="machimus_atricapillus_gca933228815v1"/>
+      <genome name="myopa_tessellatipennis_gca943737955v1"/>
+    </collection>
+
   </collections>
 
+  <multiple_alignments>
+    <!-- Drosophila pan genome -->
+    <multiple_alignment method="CACTUS_HAL" collection="pangenome_drosophila"/>
```
Comment (thiagogenez) [inline conf/metazoa/mlss_conf.xml:356]: @twalsh-ebi - maybe is it a good idea to include `url` here?

Comment (twalsh-ebi) [reply conf/metazoa/mlss_conf.xml:356]: It may be, @thiagogenez, if we can pick a filename now and stick to it. 🤔

**Comment by twalsh-ebi** [comment]
Hi @ens-LCampbell.. Sorry I didn't spot this earlier, but I've just realised that this PR has been submitted against the `release/111` branch, but it should be submitted against the `main` branch in anticipation of Compara branching to `release/112`.

**Comment by ens-LCampbell** [comment]
> Hi @ens-LCampbell.. Sorry I didn't spot this earlier, but I've just realised that this PR has been submitted against the `release/111` branch, but it should be submitted against the `main` branch in anticipation of Compara branching to `release/112`.

Ohhh excellent ! Drat ok I shall close and re make PR to main branch. Sorry Thomas

**Comment by twalsh-ebi** [comment]
> > Hi @ens-LCampbell.. Sorry I didn't spot this earlier, but I've just realised that this PR has been submitted against the `release/111` branch, but it should be submitted against the `main` branch in anticipation of Compara branching to `release/112`.
> 
> Ohhh excellent ! Drat ok I shall close and re make PR to main branch. Sorry Thomas

No worries. If it's any consolation, feedback was positive on the Compara side for replacing `octopus_bimaculoides` with `octopus_bimaculoides_gca001194135v1` in Pan Compara. If this is something that everyone would be happy with on the Metazoa side, would it be possible to include in the PR an update of this species in [line 137 of the Pan additional-species file](https://github.com/Ensembl/ensembl-compara/blob/e93c1dc2f6b017817246cdbd15baea29391281ac/conf/pan/additional_species.json#L137)?

**Pull request closed without merging**